README.md

```diff
@@ -24,6 +24,10 @@ Tab and hold the finger down on the maintenance page for 5s to hide it.
 - `androidLink`: Define an external url for Android devices. A button will be displayed that is linked to the external url.
 - `iosButtonText`: Define a text for the link on iOS devices.
 - `androidButtonText`: Define a text for the link button on Android devices.
+- `startDate`: Define a start date for the maintenance mode. You can also only define a start or an end date. Format: YYYY/MM/DD - HH:mm (e.g. `"2023/04/20 - 00:00"`).
+- `endDate`: Define a end date for the maintenance mode. You can also only define a start or an end date. Format: YYYY/MM/DD - HH:mm (e.g. `"2023/05/17 - 10:00"`).
+- `maintenancePagesWhitelist`: Whitelist that enables the maintenance mode only for configured pages (e.g. `["/cart"]`).
```

Comment:
`/item`, `/page` or `/category/373036` does not work - which pages can be added?

Reply:
reworked logic